app/v2/controllers/CreateForeignPropertyPeriodSummaryController.scala

```diff
@@ -110,9 +108,9 @@ class CreateForeignPropertyPeriodSummaryController @Inject()(val authService: En
           MtdErrorWithCode(RuleIncorrectOrEmptyBodyError.code) | RuleDuplicateSubmissionError | MtdErrorWithCode(CountryCodeFormatError.code) |
           MtdErrorWithCode(RuleCountryCodeError.code) | MtdErrorWithCode(RuleDuplicateCountryCodeError.code) | RuleTypeOfBusinessIncorrectError =>
         BadRequest(Json.toJson(errorWrapper))
-      case NotFoundError   => NotFound(Json.toJson(errorWrapper))
+      case NotFoundError => NotFound(Json.toJson(errorWrapper))
```

Comment:
Can we update this to use the new method we have been using (`    case _ \n if errorWrapper.containsAnyOf(...) => ...`)

Reply:
Updated now